insights/parsers/kdump.py

```diff
@@ -32,6 +33,7 @@ class KDumpConf(Parser):
         data (dict): a dictionary of options set in the data.
         comments(list): fully commented lines
         inline_comments(list): lines containing inline comments
+        target(tuple): target line parsed as a (x, y) tuple if set, else None
```

Comment:
The double colons "::" in line-30 must be a typo, would you please correct it in your patch?